src/models/parallelCoordinates.js

```diff
@@ -9,16 +9,19 @@ nv.models.parallelCoordinates = function() {
     // Public Variables with Default Settings
     //------------------------------------------------------------
 
-    var margin = {top: 30, right: 10, bottom: 10, left: 10}
-        , width = null
-        , height = null
+    var margin = {top: 30, right: 0, bottom: 10, left: 0}
+        , width = 1000
+        , height = 500
         , x = d3.scale.ordinal()
         , y = {}
         , dimensions = []
         , color = nv.utils.defaultColor()
         , filters = []
         , active = []
-        , dispatch = d3.dispatch('brush')
+        , foreground
+        , background
```

Comment:
foreground and background are only used inside the chart(){...} block, so just make them vars inside there.   The block of vars up here is meant to be for persistent settings.